MAUI/Pull-to-Refresh/getting-started.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Getting Started with .NET MAUI PullToRefresh control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI PullToRefresh (SfPullToRefresh) control, its elements, and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+---
+
+# Getting Started with .NET MAUI PullToRefresh (SfPullToRefresh)
+
+This section provides a quick overview of how to get started with the .NET MAUI PullToRefresh (SfPullToRefresh) for MAUI. Walk through the entire process of creating the real-world SfPullToRefresh.
+
+## Creating an application using the .NET MAUI PullToRefresh
+
+ 1. Create a new .NET MAUI application in Visual Studio.
+ 2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfPullToRefresh to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.PullToRefresh and install it.
+ 3. Import the control namespace `Syncfusion.Maui.PullToRefresh` in XAML or C# code.
+ 4. Initialize the [SfPullToRefresh]() control.
```

Comment:
for now, we can highlight "SfPullToRefresh" word. Remove the format of adding link.